### ISSUE TITLE
Can deletedgenomes or subtree be recovered?

### ISSUE THREAD

**Issue by Baohua-Chen** [ComparativeGenomicsToolkit/hal#289]
Hi,
I have deleted some genomes using halRemoveSubtree from a hal file. I didn't realize that it will change the input file directly rather than generate a new file at the time.
I noticed that the new file have similar size to the original one. So, Is it posible to recover the deleted genomes or subtree?

Thanks,

Chen

**Comment by glennhickey** [comment]
I don't think so, unfortunately.  The information is still in the file in theory, but it's been unlinked in a way that I don't think HAL can recover it from.

**Comment by Baohua-Chen** [comment]
Thanks very much for your reply. @glennhickey
